### DISCUSSION TITLE
[feature]: 全文检索功能

### DISCUSSION THREAD

**Discussion by ctxqlxs** [ling7334/hoppscotch-backend-go#7] [Ideas]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current behavior

现在只能检索标题

### Steps to reproduce

1

### Environment

Release

### Version

Self-hosted

**Comment by ling7334** [comment]
搜索目前是纯前端实现的，hoppscotch源项目目前好像有个PR在做，但还没merge，而且也是按标题搜索
[feat: full text search for TeamCollections and TeamRequests](https://github.com/hoppscotch/hoppscotch/pull/3857)

**Comment by ling7334** [comment]
hoppscotch数据库用的postgres，
想来要做全文搜索，要么加倒排索引，要不然就要加搜索引擎组件了（elasticsearch，meilisearch，manticore，etc...）,但这样就有点费劲了，而且更复杂了。我估计hoppscotch组不太会做。

**Comment by ling7334** [comment]
最新的2024.3.0版合入了，我修改看下

**Comment by ling7334** [comment]
在[15896c1](https://github.com/ling7334/hoppscotch-backend-go/commit/15896c1ac582e5255b10a3ea53a7f9e89a9c454f)加了个endpoint，但hoppscotch还是只搜标题，没搜内容。
